### PR TITLE
CEENG 222 UI

### PR DESCRIPTION
**Pull request by nikitasheremet**
Added Image Carousel Component

image

### CODE REVIEW

File: src/Containers/ImageCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import { MainInterface, ResponsiveInterface } from '@Utils/BaseStyles';
+import { ImplicitPropsInterface } from '@Utils/Hooks';
+
+export interface ImageCarouselProps
+    extends MainInterface,
+        ResponsiveInterface,
+        ImplicitPropsInterface {
+    imageData: string[];
+    onClick: Function;
+    altText: string;
```
Comment (ralph-dev) [inline src/Containers/ImageCarousel.tsx]: Doesn’t this need to be an array?

File: src/Containers/ImageCarousel.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        <Items {...props}>
+            {imageData.map(image => (
+                <Item key={image} onClick={() => onClick(image)}>
+                    <Overlay>
+                        <Icon /> Delete
+                    </Overlay>
+                    <img
+                        width={width}
+                        height={height}
+                        alt={altText}
```
Comment (ralph-dev) [inline src/Containers/ImageCarousel.tsx]: This will have the same alt for every image is that right?

File: src/Containers/ImageCarousel.tsx (hunk 1 above)
Comment (nikitasheremet) [reply src/Containers/ImageCarousel.tsx]: It is? It is an array of strings

File: src/Containers/ImageCarousel.tsx (hunk 2 above)
Comment (nikitasheremet) [reply src/Containers/ImageCarousel.tsx]: Yeah I was thinking it would just say either Store Image, or Store Logo

**Pull request merged**